pkgs/goreleaser/goreleaser/registry.yaml

```diff
@@ -4,24 +4,55 @@ packages:
     repo_owner: goreleaser
     repo_name: goreleaser
     description: Deliver Go binaries as fast and easily as possible
-    replacements:
-      amd64: x86_64
-    asset: goreleaser_{{title .OS}}_{{.Arch}}.{{.Format}}
-    format: tar.gz
-    version_constraint: semver(">= 0.182.0")
-    overrides:
-      - goos: darwin
-        asset: goreleaser_{{title .OS}}_all.tar.gz
-      - goos: windows
-        format: zip
+    version_constraint: "false"
     version_overrides:
-      # https://github.com/goreleaser/goreleaser/pull/2572
-      # macOS Universal binaries
+      - version_constraint: semver("< 0.182.0")
+        asset: goreleaser_{{title .OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        replacements:
+          amd64: x86_64
+        checksum:
+          type: github_release
+          asset: checksums.txt
+          algorithm: sha256
+        overrides:
+          - goos: windows
+            format: zip
+      - version_constraint: semver("<= 1.2.1")
+        asset: goreleaser_{{title .OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        replacements:
+          amd64: x86_64
+        checksum:
+          type: github_release
+          asset: checksums.txt
+          algorithm: sha256
+        overrides:
+          - goos: darwin
+            asset: goreleaser_{{title .OS}}_all.{{.Format}}
+          - goos: windows
+            format: zip
       - version_constraint: "true"
+        asset: goreleaser_{{title .OS}}_{{.Arch}}.{{.Format}}
+        format: tar.gz
+        replacements:
+          amd64: x86_64
+        checksum:
+          type: github_release
+          asset: checksums.txt
+          algorithm: sha256
+          cosign:
+            opts:
+              - --certificate-identity
+              - https://github.com/goreleaser/goreleaser/.github/workflows/release.yml@refs/tags/{{.Version}}
```

Comment:
This differs a from most existing cosign configs by using `--certificate-identity` (not `--certificate-identity-regexp`) and by using the exact identity of the release. Wondering if this is ok, or if there's a specific reason to use the regexp version and not being this strict?

Reply:
No problem.
I don't remember why we used `--certificate-identity-regexp`, but I think we should use `--certificate-identity` as much as possible.

Reply:
I remenber why we use `--certificate-identity-regexp`.
If GitHub Actions Reusable Workflows are used, the certificate-identity is the reusable workflow's URL such as `https://github.com/suzuki-shunsuke/go-release-workflow/.github/workflows/release.yaml@v0.1.0`.
I didn't want to fix the setting every time the reusable workflow is updated, so I used regular expressions `@.*`.
But if reusable workflows aren't used, I think `--certificate-identity` is better.